cli/src/torii/routing.rs

```diff
@@ -478,11 +476,11 @@ impl Torii {
     ) -> impl warp::Filter<Extract = impl warp::Reply> + Clone + Send {
         let get_router_status = endpoint2(
```

Comment:
This is the culprit. The `2` here is the number of arguments. You need to replace it with the inline code that would have been generated for `1` (we don't have an `endpoint1` macro).